core/domain/collection_domain.py

```diff
@@ -321,13 +322,13 @@ def from_dict(
         """Return a Collection domain object from a dict.
 
         Args:
-            collection_dict: dict. The dictionary representation of  the
-                collection.
-            collection_version: int. The version of the collection.
+            collection_dict: dict. The dictionary representation of the
+                Collection.
```

Comment:
You don't need to capitalize these; docstring descriptions should be written in regular prose.